canonical-web-and-design/charmhub.io
# Cleanup filters.js

## Done

- Moved `buildCharmCard` to it's own file
- Removed commented out code
- Removed unnecessary `console.log` statements
- Removed unused code

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the site looks and works the same as it does on production
